src/bitmessagekivy/tests/test_setting_screen.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=too-few-public-methods
+
+from .telenium_process import TeleniumTestProcess
+from .common import skip_screen_checks
+
+
+class SettingScreen(TeleniumTestProcess):
+    """Setting Screen Functionality Testing"""
+
+    @skip_screen_checks
+    def test_setting_screen(self):
+        """Show Setting Screen"""
+        print("=====================Test -Show Setting Screen=====================")
+        self.cli.sleep(3)
+        # this is for checking current screen
+        self.assertExists("//Inbox[@name~=\"inbox\"]", timeout=2)
+        # this is for opening Nav drawer
+        self.cli.wait_click('//MDActionTopAppBarButton[@icon=\"menu\"]', timeout=2)
+        # checking state of Nav drawer
+        self.assertExists("//MDNavigationDrawer[@state~=\"open\"]", timeout=2)
+        # this is for scrolling Nav drawer
+        self.drag("//NavigationItem[@text=\"Sent\"]", "//NavigationItem[@text=\"Inbox\"]")
+        # assert for checking scroll funcation
+        scroll_distance = self.cli.getattr('//ContentNavigationDrawer//ScrollView[0]', 'scroll_y')
+        self.assertCheckScrollDown(scroll_distance, -0.0, timeout=1)
```

Comment:
Here, `scroll_distance` should be a reference as otherwise the test is not very helpful.

Reply:
Here, you retrieve `scroll_distance`, which I presume is a float, and then pass it to the assert. Instead, it should pass some sort of reference.